angucomplete-alt.js

```diff
@@ -319,12 +321,18 @@
         }
 
         function dropdownRowOffsetHeight(row) {
+          if (scope.preventCssComputation) {
```

Comment:
IT seems for this an below you wouldnt call the function if value is set. Can we add the check to the caller of these functions instead of returning 0?